shared/templates/sysctl/bash.template

```diff
@@ -22,6 +22,16 @@ for f in /etc/sysctl.d/*.conf /run/sysctl.d/*.conf /usr/local/lib/sysctl.d/*.con
   fi
 done
 
+#
+# Set sysctl config file which to save the desired value
+#
+{{% if product in [ "sle12", "sle15"] %}}
```

Comment:
> Could you be so kind and create a property instead listing products? Something like was done for `sshd_distributed_config`: https://github.com/search?q=org%3AComplianceAsCode+sshd_distributed_config&type=code
> 
> These properties can make controlling rule behavior easier than listing products. We have plans to make these properties more evident and better documented.

Thanks @yuumasato for the feedback. Still I am not sure I understand the property suggestion. Is it to have a property for the default config `/etc/sysctl.conf` and if not defined to use per var file?

Regarding ansible, it wasn't in my initial plans, but I guess it will be better both to behave consistently, so will add `sysctl_file` param to the sysctl module there.

Reply:
> > Could you be so kind and create a property instead listing products? Something like was done for `sshd_distributed_config`: https://github.com/search?q=org%3AComplianceAsCode+sshd_distributed_config&type=code
> > These properties can make controlling rule behavior easier than listing products. We have plans to make these properties more evident and better documented.
> 
> Thanks @yuumasato for the feedback. Still I am not sure I understand the property suggestion. Is it to have a property for the default config `/etc/sysctl.conf` and if not defined to use per var file?

Yes, I think we are talking about the same thing. This property would control whether the template remediates into the drop-in files or not.
We could, let say, define `sysctl_remediate_drop_in_file`, and then in the product's `product.yml`, define whether we want the product to remediate sysctl in drop-in files or not.

See for example: https://github.com/ComplianceAsCode/content/blob/master/products/rhel9/product.yml#L22

> Regarding ansible, it wasn't in my initial plans, but I guess it will be better both to behave consistently, so will add `sysctl_file` param to the sysctl module there.

:)

Reply:
It would be very much appreciated if you add this property into the project, but I understand if you don't feel like it.